docs/user-guide/building-assistants.rst

```diff
@@ -155,9 +155,9 @@ For example:
 
 
 While it's good to test the bot interactively, we should also add end to end test cases that
-can later be included as part of our CI/CD system. `End to end stories <https://rasa.com/docs/rasa/user-guide/testing-your-assistant/#end-to-end-testing>`_
+can later be included as part of our CI/CD system. End-to-end :ref:`test conversations <end-to-end-testing>`
 include NLU data, so that both components of Rasa can be tested.  Create a file called
-``test_stories.md`` in the root directory with some test cases:
+``test_stories.md`` in a subdirectory `tests` with some test cases:
 
```

Comment:
Did we start with `rasa init` here? If so we should be editing the file in the `tests` folder instead of creating a new one

Reply:
True

Reply:
It's also prepolated with test conversations

Reply:
Made a new comment